internals/overlord/logstate/syslog/syslog.go

```diff
@@ -155,7 +155,7 @@ func (c *Client) Add(entry servicelog.Entry) error {
 	entry.Message = strings.TrimSuffix(entry.Message, "\n")
 
 	c.entries = append(c.entries, entryWithService{
-		timestamp: entry.Time.Format(time.RFC3339Nano), // Format: 2021-05-26T12:37:01.123456789Z
+		timestamp: entry.Time,
```

Comment:
I think at this point we can just get rid of `entryWithService` and use a `[]servicelog.Entry` type directly, as they have exactly the same fields.